apps/browser/lib/hooks/useEvoService.ts

```diff
@@ -129,6 +132,10 @@ export const useEvoService = (
       return [];
     }
 
+    if (currentChat.title) {
+      setCurrentChatInfo({ name: currentChat.title })
```

Comment:
This isn't necessary since we're setting chat title in the migration for old chats that do not have a title.